logs-lib/logs/main.libsonnet

```diff
@@ -0,0 +1,42 @@
+local dashboards = import './dashboards.libsonnet';
+local panels = import './panels.libsonnet';
+local targets = import './targets.libsonnet';
+local variables = import './variables.libsonnet';
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+
+{
+  new(
```

Comment:
as i mentioned above, a `new()` method creates a new **something**. We want to make sure that the something is clear. E.g. making a new logs will confuse me. If you said "make me a new logs dashboard" I'd understand better.